source/simulator/helper_functions.cc

```diff
@@ -215,6 +223,71 @@ namespace aspect
   }
 
 
+
+  template <int dim>
+  void Simulator<dim>::write_plugin_graph (std::ostream &out) const
+  {
+    // write the preamble
+    out << "digraph Plugins\n"
+        "{\n"
+        "  splines=line;\n"
+        "  splines=true;\n"
```

Comment:
Good question, I don't know either, but it makes a difference and that's what the online documentation also seems to suggest one uses. Don't ask me...